src/gui/tray/ActivityListModel.h

```diff
@@ -78,22 +84,28 @@ class ActivityListModel : public QAbstractListModel
     Q_INVOKABLE void triggerDefaultAction(int activityIndex);
     Q_INVOKABLE void triggerAction(int activityIndex, int actionIndex);
 
+    AccountState *accountState() const;
+
 public slots:
     void slotRefreshActivity();
     void slotRemoveAccount();
 
-private slots:
-    void slotActivitiesReceived(const QJsonDocument &json, int statusCode);
-
 signals:
     void activityJobStatusCode(int statusCode);
     void sendNotificationRequest(const QString &accountName, const QString &link, const QByteArray &verb, int row);
 
 protected:
+    void slotActivitiesReceived(const QJsonDocument &json, int statusCode);
```

Comment:
is this a slot ?
the name  implies it but the lack of `slot` keyword confuses me

Reply:
It is used in as a connection to a signal, but I think there is no need to make it a slot. I will change the name.